wagn/lib/decko/engine.rb

```diff
@@ -35,13 +34,6 @@ class Engine < ::Rails::Engine
         
         ActiveRecord::Base.establish_connection(::Rails.env)
       end
-      ActiveSupport.on_load(:after_initialize) do
-        begin
-          require_dependency 'card' unless defined?(Card)
-        rescue ActiveRecord::StatementInvalid => e
-          ::Rails.logger.warn "database not available[#{::Rails.env}] #{e}"
-        end
-      end
```

Comment:
I don't really like this code.  Gerry, I'm guessing there was some reason for it, but:
a) this is an odd case to catch a missing database, and
b) card will soon be required if a controller action is triggered.

I would suggest that we move this kind of check into cardio.